SingularityBase/src/main/java/com/hubspot/mesos/SingularityContainerInfo.java

```diff
@@ -18,6 +19,8 @@ public SingularityContainerInfo(
       @JsonProperty("type") SingularityContainerType type,
```

Comment:
I could see an argument for making this one `Optional<SingularityContainerType>` with a sane default